tests/www/views/test_views_connection.py

```diff
@@ -171,6 +171,7 @@ def test_duplicate_connection_error(admin_client):
     with create_session() as session:
         session.query(Connection).delete()
         session.add_all(connections)
+        session.commit()
```

Comment:
This should not be needed because `create_session` commits automatically on exit.